geppetto.js/geppetto-ui/src/3d-canvas/threeDEngine/ThreeDEngine.js

```diff
@@ -33,6 +33,7 @@ export default class ThreeDEngine {
     selectionStrategy
   ) {
     this.scene = new THREE.Scene();
+    this.backgroundColor = backgroundColor;
```

Comment:
why do we need to store this, it's already in the `this.scene.background`, we can use this in the `update` function to test against.